src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+
+function CapitalizeAndSpace(string) {
```

Comment:
```suggestion
function capitalizeAndSpace(string) {
```
camelCase for common functions (it's not constructor function)